Webtest.sh

```diff
@@ -38,10 +40,12 @@ if [ "$SIZE" -lt 50000 ]; then
 fi
 # Check the main page
 $CURL $BASE_FORTIO
-# Do a small load using std client
+# Do a small http load using std client
 docker exec fortio_server /usr/local/bin/fortio load -stdclient -qps 1 -t 2s -c 1 https://www.google.com/
 # and with normal and with custom headers
 docker exec fortio_server /usr/local/bin/fortio load -H Foo:Bar -H Blah:Blah -qps 1 -t 2s -c 2 http://www.google.com/
 # Do a grpcping
 docker exec fortio_server /usr/local/bin/fortio grpcping localhost
+# Do a grpcping to a scheme-prefixed destination
+docker exec fortio_server /usr/local/bin/fortio grpcping https://fortio.istio.io:443
```

Comment:
👍 can we make it work without :443 :-) ? [it's ok as, we can do that later]

Reply:
you mean, be intelligent and assume 443 on a https prefix?

Reply:
yes if no :port is specified but http:// prefix is there, use 80, if https:// is there, use 443
but we can do that in followup

Reply:
opened 162 to track that